app/models/participant_profile/ecf.rb

```diff
@@ -98,36 +96,18 @@ def policy_class
     end
 
     def relevant_induction_record(lead_provider:)
```

Comment:
could we consolidate `relevant_induction_record` and `latest_induction_record_for`, I think the only difference is the cohort, but does it matter 🤔

Reply:
checking prod we have no induction records in cohort 2020, so maybe we can always use the cohort filter?

Reply:
Yes nothing relevant in 2020, I looked before starting this. I've changed the internal calls in `ecf.rb` to use `relevant_induction_record` instead of ` latest_induction_record_for` but kept that interface as it looks like finance stuff uses it.